backend/database/migrations/2021_11_22_050755_create_follows_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateFollowsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('follows', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->constrained()->cascadeOnDelete();
+            $table->integer('follow_id');
```

Comment:
Need pd nino i foreignId c `follow_id` and cascadeOnDelete para if ma remove imo g follow na user id ma remove ra ni nga relationship.

Reply:
Ok sir, noted on this

Reply:
pd ra nimo ni i add sa last.
```
$table->foreign('follow_id')
    ->references('id')
    ->on('users')
    ->onDelete('cascade');
```